src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(currentNum) {
+  let currentNummount = currentNum || 0;
```

Comment:
There is a typo in the variable name `currentNummount`. It should be `currentNum` to match the intended logic and avoid confusion.